docs/customization/custom-scripts.md

```diff
@@ -140,6 +140,8 @@ The Script class provides two convenience methods for reading data from files:
 
 These two methods will load data in YAML or JSON format, respectively, from files within the local path (i.e. `SCRIPTS_ROOT`).
 
+**Note:** These convenience methods only work if running scripts within the local path, they will not work if running scripts using AWS S3.
```

Comment:
Is this limitation unique to S3, or does it apply to _any_ external storage?

Reply:
As per discussion this will be deprecated.